parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -206,33 +206,21 @@ function setDockerOpts () {
 function configAzureNetworkPolicy() {
     CNI_CONFIG_DIR=/etc/cni/net.d
     mkdir -p $CNI_CONFIG_DIR
-
     chown -R root:root $CNI_CONFIG_DIR
     chmod 755 $CNI_CONFIG_DIR
-
-    # Download Azure VNET CNI plugins.
```

Comment:
cleaning up comments while I pass through regions of code to minimize CSE payload